data/src/main/java/org/oppia/data/backends/gae/model/Topic.kt

```diff
@@ -0,0 +1,25 @@
+package org.oppia.data.backends.gae.model
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class Topic(
+
+  /*
+   * Ignore:
+   * is_moderator, is_admin, username, user_email, iframed, is_topic_manager, is_super_admin
+   */
+
+  @Json(name = "topic_id") val topicId: String?,
+  @Json(name = "topic_name") val topicName: String?,
+  @Json(name = "canonical_story_dicts") val canonicalStoryDicts: List<CanonicalStorySummary?>?,

```

Comment:
For lists & collections of objects, can they actually be absent?

Reply:
I have seen responses in which lists and collections are also absent as well as null.